CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - DOC: Convert CompareFlights example notebooks to .rst files [#576](https://github.com/RocketPy-Team/RocketPy/pull/576)
 - MNT: Refactor inertia calculations using parallel axis theorem [#573] (https://github.com/RocketPy-Team/RocketPy/pull/573)
 - ENH: Optional argument to show the plot in Function.compare_plots [#563](https://github.com/RocketPy-Team/RocketPy/pull/563)
+- BLD: Change setup.py to pyproject.toml [#589](https://github.com/RocketPy-Team/RocketPy/pull/589)
```

Comment:
Sorry to be pedantic, but this line appears to be out of order.

Reply:
Oopsie, fixed!